### PR TITLE
update links

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/nunit/docs/issues/637

### CODE REVIEW

File: docs/articles/developer-info/Packaging-the-V2-Adapter.md [hunk 1]
```diff
@@ -67,13 +67,13 @@ The resulting files can be found in the "package" folder:
...
-You need this to work with the vsix.  Download from <http://www.microsoft.com/en-us/download/details.aspx?id=40758>
+You need this to work with the vsix.  Download from <https://www.microsoft.com/en-us/download/details.aspx?id=40758>
 
 1. **NAnt**
-Download from <http://nant.sourceforge.net/>.  We use the 0.92 version.
+Download from <https://nant.sourceforge.net/>.  We use the 0.92 version.
 
 1. **NuGet**
-You need the nuget.exe in your path.  Download the exe from <http://nuget.codeplex.com/downloads/get/784779>.  We use the 2.8 version
+You need the nuget.exe in your path.  Download the exe from <https://nuget.codeplex.com/downloads/get/784779>.  We use the 2.8 version
```
Comment (SeanKilleen) [inline docs/articles/developer-info/Packaging-the-V2-Adapter.md:76]: @Arhell are you personally checking these links to ensure they have `https` editions? This link here, for example, doesn't appear to work anymore at all, so I'm not sure it makes sense to change to https.

Comment (Arhell) [reply docs/articles/developer-info/Packaging-the-V2-Adapter.md]: All check & update

File: docs/articles/developer-info/Packaging-the-V2-Adapter.md [hunk 2]
```diff
@@ -67,13 +67,13 @@ The resulting files can be found in the "package" folder:
...
 1. **Visual Studio 2013 SDK**  
-You need this to work with the vsix.  Download from <http://www.microsoft.com/en-us/download/details.aspx?id=40758>
+You need this to work with the vsix.  Download from <https://www.microsoft.com/en-us/download/details.aspx?id=40758>
 
 1. **NAnt**
 Download from <http://nant.sourceforge.net/>.  We use the 0.92 version.
 
 1. **NuGet**
-You need the nuget.exe in your path.  Download the exe from <http://nuget.codeplex.com/downloads/get/784779>.  We use the 2.8 version
+You need the nuget.exe in your path.  Download the exe fro<http://nuget.codeplex.com/downloads/get/784779>.  We use the 2.8 version
```
Comment (SeanKilleen) [inline docs/articles/developer-info/Packaging-the-V2-Adapter.md:76]:
```suggestion
You need the nuget.exe in your path.  Download the exe from <http://nuget.codeplex.com/downloads/get/784779>.  We use the 2.8 version
```

File: docs/articles/nunit/technical-notes/nunit-internals/NUnit-3.0-Architecture-(2009).md [hunk 3]
```diff
@@ -13,7 +13,7 @@ This specification describes the overall layered architecture of the NUnit Platf
 
 ### Launchpad Blueprint
 
-* <http://blueprints.launchpad.net/nunit-3.0/+spec/nunit-3.0-architecture>
+* <https://blueprints.launchpad.net/nunit-3.0/+spec/nunit-3.0-architecture>
```
Comment (SeanKilleen) [inline docs/articles/nunit/technical-notes/nunit-internals/NUnit-3.0-Architecture-(2009).md:16]: This page gets a 404 error. I don't believe we should update the URL if we don't know it exists.

**Review by SeanKilleen — CHANGES_REQUESTED** [review]

**Comment by Arhell** [comment]
done

**Review by SeanKilleen — APPROVED** [review]
Looks great 👍 thanks for this!

**Pull request merged**